client/src/components/event/results.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { CardDeck, Container } from 'react-bootstrap';
+import ButtonToolbar from 'react-bootstrap/ButtonToolbar';
+import Spinner from 'react-bootstrap/Spinner';
+import CardView from './card-view';
+
+class Results extends Component {
+  render() {
+    const {
+      isFetching, events, eventContainerClassName,
+      cardDeckClassName, spinnerContainerClassName,
+      buttonToolbarClassName, spinnerAnimation,
+      spinnerVariant,
+    } = this.props;
+    return (
+      <Container className={eventContainerClassName}>
+        {isFetching ? (
+          <CardDeck className={cardDeckClassName}>
+            {events.map((event) => (
+              <CardView key={event.id} event={event} isFetching={isFetching} />
+            ))}
+          </CardDeck>
+        ) : (
+          <Container className={spinnerContainerClassName}>
+            <ButtonToolbar className={buttonToolbarClassName}>
+              <Spinner animation={spinnerAnimation} variant={spinnerVariant} />
+            </ButtonToolbar>
+          </Container>
+        )}
```

Comment:
This ternar operator is looked a little big. https://github.com/airbnb/javascript#comparison--nested-ternaries